bundles/net.tourbook/src/net/tourbook/ui/views/rawData/DialogMergeTours.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (C) 2005, 2021 Wolfgang Schramm and Contributors
+ * Copyright (C) 2005, 2022 Wolfgang Schramm and Contributors
  *
```

Comment:
The elevation diff line, the most important line when adjusting the time, is truncated when changing the sec values for the horizontal adjustment

**Old code**

![image](https://user-images.githubusercontent.com/1283445/151691563-5f9f08c5-3560-438a-b10f-fb46eb461b15.png)

**New code**

![image](https://user-images.githubusercontent.com/1283445/151691648-b46f8a51-8052-4c85-ac83-aee03dc46002.png)

Reply:
It seems that this 12 year old code is still useful :-)

Reply:
Yes, I have some old tours that I want to merge. I discovered this feature but it was lacking the speed merging

Reply:
Setting the PR as draft as I am investigating and fixing the issues/regressions you found.

Reply:
Fixed

![image](https://user-images.githubusercontent.com/25235665/152876347-ba5eb26a-c7af-4e34-b2f7-a3e57393b578.png)